xkcd_dl/annotated.py

```diff
@@ -1,6 +1,6 @@
 #!/usr/bin/env python3
```

Comment:
As discussed in #22 . This file is just for reference. The main endpoint would be the `cli.py` file